CourseManager/Assets.xcassets/Colors-1/Background 2.colorset/Contents.json

```diff
@@ -0,0 +1,72 @@
+{
```

Comment:
@sghourab try to avoid using numbers, spaces in naming for assets and classes. Think if you can rename them in more clear way